### DISCUSSION TITLE
Creating workspace left or right of current workspace

### DISCUSSION THREAD

**Discussion by tiredsapiens** [i3/i3#6146] [Q&A] [answered]
hello , i have somewhat of a little annoyance when using i3, sometimes i need to create workspace 6 7 8 and the only way to do it ( that i know of) is to press $mod + that number ,which can be slightly uncomfortable when the number is high enough. is there a way/command to create a workspace left or right to current workspace? for example if im on workspace 4 already and i create to right of it , i create workspace 5 , and  if im on workspace 5 i create workspace 6.

**Comment by orestisfl** [comment, marked as answer]
This has been asked a few times, I made a post perhaps making it easier for the next people to find: https://github.com/i3/i3/discussions/6147
